sbi/inference/posteriors/importance_posterior.py

```diff
@@ -179,22 +181,25 @@ def sample(
                 sample_shape,
                 oversampling_factor=oversampling_factor,
                 max_sampling_batch_size=max_sampling_batch_size,
+                show_progress_bars=show_progress_bars,
             )
         elif self.method == "importance":
-            return self._importance_sample(sample_shape)
+            return self._importance_sample(sample_shape, show_progress_bars=show_progress_bars)
```

Comment:
please remove passing this argument. you would have to implement the progress bar in the downstream call in `importance_sample(...)` (see comment below). But actually, the sampling with this function happens basically instantly, so no progress bar needed here.